config/locales/en/publisher_information/publisher_updates.yml

```diff
@@ -0,0 +1,55 @@
+en:
+  publisher_information:
+    publisher_updates:
+      title: What’s new in Whitehall Publisher
+      summary: |
+        Summary of updates to Whitehall Publisher, content design guidance, or the design of GOV.UK.
+      last_updated: Last updated 5 September 2022
+      intro_section:
+        heading: Moving to the Design System
+        body_govspeak: |
+          Whitehall Publisher will be moving from the current styling to use the styling and components of the [GOV.UK Design System](https://design-system.service.gov.uk/).
+          This will make Whitehall Publisher more accessible, more user friendly and give you to have a consistent user experience throughout your publishing journey on GOV.UK.
+
+          [Read more about the changes on the content community Basecamp](https://3.basecamp.com/4322319/buckets/15005645/messages/5287245970).
+      sections:
+        - heading: Recent changes
```

Comment:
Are we expecting there to be multiple sections? I'd guess this page is only to show the one section "Recent changes" ?

Reply:
Yes, there's another section that's not available at the moment called `Upcoming changes`. I have also moved all the other sections into one.

Reply:
Cool - rather than having a concept of sections shall we just name the bits recent_changes and upcoming_changes. We should be very cautious with making locale files flexible since they're not designed as a makeshift CMS and should veer towards rigid.

Reply:
This has been done